assets/src/edit-story/components/panels/stylePreset/test/panel.js

```diff
@@ -204,7 +201,7 @@ describe('Panels/StylePreset', () => {
 
       getTextPresets.mockImplementation(() => {
         return {
-          textStyles: [STYLE_PRESET],
+          textStyles: [{ color: TEST_COLOR_2, ...STYLE_PRESET }],
```

Comment:
Same here as below:
Just a color alone would not be part of textStyles, it would be part of textColors.